packages/editor/src/editor/canvas/useComponentBlockActions.ts

```diff
@@ -117,7 +118,10 @@ export const useComponentBlockActions = ({
     }
     if (e.code === 'KeyE' && config.quickActions.find(q => q === 'EXTRACTINTOCOMPONENT')) {
       e.stopPropagation();
-      setTimeout(() => setShowExtractDialog(true), 0);
+      setTimeout(() => {
+        activateLocalScopes();

```

Comment:
where is the reset? I would handle this the same way as the others... onOpenChange